notation/verify.ts

```diff
@@ -0,0 +1,41 @@
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { getArtifactReferences } from './lib/variables';
+import { getConfigHome } from './lib/fs';
+
+export async function verify(): Promise<void> {
+    const artifactRefs = getArtifactReferences();
+    const trustpolicy = taskLib.getInput('trustpolicy', true) || '';
+    const truststore = taskLib.getInput('truststore', false) || '';
+    const configHome = getConfigHome();
+    const allowReferrerAPI = taskLib.getBoolInput('allowReferrersAPI', false);
+    const debug = taskLib.getVariable('system.debug')
+
+    // copy the trustpoliy to notation config dirctory
+    const trustPolicyPath = path.join(configHome, 'notation', 'trustpolicy.json');
+    taskLib.cp(trustpolicy, trustPolicyPath, '-f');
```

Comment:
`notation policy import` should be used here.
On succeed, use `notation policy show` to output the policy to console for user's reference.

Reply:
Updated.